cinema/cinema/Controllers/SeatController.cs

```diff
@@ -1,48 +1,50 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
-using Cinema.DataAccess;
+using Cinema.Domain.Interfaces;
 using Cinema.Domain.Models;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
 
 namespace Cinema.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
-    public class SeatController : Controller
+    public class SeatController : Controller, ISeatsService
```

Comment:
ISeatsService este o interfata care este implementata deja in SeatsService. In controller noi injectam SeatsService, nu avem de ce sa implementam ISeatsService;
la fel si in celelalte controlere

Reply:
solved